bundles/org.openhab.binding.juicenet/src/main/java/org/openhab/binding/juicenet/internal/handler/JuiceNetBridgeHandler.java

```diff
@@ -0,0 +1,178 @@
+/**
+ * Copyright (c) 2010-2022 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.juicenet.internal.handler;
+
+import static org.openhab.binding.juicenet.internal.JuiceNetBindingConstants.DEVICE_THING_TYPE;
+
+import java.io.IOException;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.eclipse.jetty.client.HttpClient;
+import org.openhab.binding.juicenet.internal.api.JuiceNetApi;
+import org.openhab.binding.juicenet.internal.api.JuiceNetApiException;
+import org.openhab.binding.juicenet.internal.api.dto.JuiceNetApiDevice;
+import org.openhab.binding.juicenet.internal.config.JuiceNetBridgeConfiguration;
+import org.openhab.binding.juicenet.internal.discovery.JuiceNetDiscoveryService;
+import org.openhab.core.thing.Bridge;
+import org.openhab.core.thing.ChannelUID;
+import org.openhab.core.thing.Thing;
+import org.openhab.core.thing.ThingStatus;
+import org.openhab.core.thing.ThingStatusDetail;
+import org.openhab.core.thing.binding.BaseBridgeHandler;
+import org.openhab.core.thing.binding.ThingHandlerService;
+import org.openhab.core.types.Command;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link JuiceNetBridgeHandler} is responsible for handling commands, which are
+ * sent to one of the channels.
+ *
+ * @author Jeff James - Initial contribution
+ */
+@NonNullByDefault
+public class JuiceNetBridgeHandler extends BaseBridgeHandler {
+
+    private final Logger logger = LoggerFactory.getLogger(JuiceNetBridgeHandler.class);
+
+    protected JuiceNetBridgeConfiguration config = new JuiceNetBridgeConfiguration();
+    private final JuiceNetApi api;
+
+    public JuiceNetApi getApi() {
+        return api;
+    }
+
+    protected @Nullable ScheduledFuture<?> pollingJob;
+    protected List<JuiceNetApiDevice> listDevices = Collections.<JuiceNetApiDevice> emptyList();
```

Comment:
This seems used only locally in `goOnline`, so can be removed as instance variable.

Reply:
Correct, however, I am fixing the issue where the binding didn't support manual configuration and need to store the info here since the token is required for the device.  Once I get this fix checked in, let's review this one again.